extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.tsx

```diff
@@ -170,22 +156,42 @@ function TrackedCornerstoneViewport(props) {
     };
   }, [measurementService, sendTrackedMeasurementsEvent, viewportId, viewportGridService]);
 
-  function switchMeasurement(direction) {
-    const newTrackedMeasurementUID = _getNextMeasurementUID(
-      direction,
-      servicesManager,
-      trackedMeasurementUID,
-      trackedMeasurements
-    );
+  const switchMeasurement = useCallback(
+    direction => {
+      const newTrackedMeasurementUID = _getNextMeasurementUID(
+        direction,
+        servicesManager,
+        trackedMeasurementUID,
+        trackedMeasurements
+      );
 
-    if (!newTrackedMeasurementUID) {
-      return;
-    }
+      if (!newTrackedMeasurementUID) {
+        return;
+      }
 
-    setTrackedMeasurementUID(newTrackedMeasurementUID);
+      setTrackedMeasurementUID(newTrackedMeasurementUID);
 
-    measurementService.jumpToMeasurement(viewportId, newTrackedMeasurementUID);
-  }
+      measurementService.jumpToMeasurement(viewportId, newTrackedMeasurementUID);
+    },
+    [measurementService, servicesManager, trackedMeasurementUID, trackedMeasurements, viewportId]
+  );
+
+  useEffect(() => {
+    const statusComponent = _getStatusComponent(isTracked);
+    const arrowsComponent = _getArrowsComponent(isTracked, switchMeasurement);
+
+    viewportActionCornersService.setActionComponent({
+      viewportId,
+      componentId: 'viewportStatusComponent',
+      component: statusComponent,
+    });
+
+    viewportActionCornersService.setActionComponent({
+      viewportId,
+      componentId: 'viewportActionArrowsComponent',
+      component: arrowsComponent,
+    });
```

Comment:
I'm kind of confused with the API. Who determines where (topLeft, etc.) to place this component?